core/pom.xml

```diff
@@ -141,6 +141,56 @@
     </dependencies>
 
     <build>
+        <!--
+Using filtering means we have to explicitly control Resources for Test and production..
+ -->
+        <resources>
+            <resource>
+                <directory>src/main/resources</directory>
+                <includes>
+                    <include>application.properties</include>
+                </includes>
+                    <excludes>
+                        <exclude>static</exclude>
+                        <exclude>template</exclude>
+                        <exclude>**/*.sql</exclude>
+                        <exclude>**/*.yaml</exclude>
+                    </excludes>
+                <filtering>true</filtering>
+            </resource>
+            <resource>
+                <directory>src/main/resources</directory>
+                <excludes>
+                    <exclude>application.properties</exclude>
+                </excludes>
+                <includes>
+                    <include>**</include>
+                </includes>
+                <filtering>false</filtering>
+            </resource>
+        </resources>
+        <testResources>
+            <testResource>
+                <directory>src/test/resources</directory>
+                <includes>
+                    <include>*</include>
+                </includes>
+                <filtering>true</filtering>
+            </testResource>
+            <testResource>
+                <directory>src/main/resources/</directory>
+                <includes>
+                    <include>application.properties</include>
+                </includes>
+                <excludes>
+                    <exclude>static</exclude>
+                    <exclude>template</exclude>
+                    <exclude>**/*.sql</exclude>
+                    <exclude>**/*.yaml</exclude>
```

Comment:
I found that using * or ** to indicate exclude all except for what was in the include caused maven to get confused and the application.properties would not be transferred over. 
I decided it was better to go with the explicit approach as it would show exactly what was being moved.

I also wanted to ensure that filtering was not being executed against the html or js as it is possible that something that looks like a marker "@something.marker@" might exist there in auto generated code or other code I am unfamiliar with. So I wanted to be especially explicit about the static and template folders.